circle.yml

```diff
@@ -30,10 +30,12 @@ jobs:
     ##
 
     - run:
-        name: Update PATH and Define Environment Variable at Runtime
+        name: Update PATH and Define Environment Variables at Runtime
         command: |
+          echo 'export GOROOT=$HOME/go' >> $BASH_ENV
```

Comment:
Why do you need `GOROOT` here?

Reply:
Because now I leave the downloaded go in `$HOME/go` instead of `sudo mv`ing it to `/usr/local` (seems cleaner to me).